crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -58,17 +70,27 @@ impl DynamicScene {
         scene
     }
 
+    /// Write the dynamic entities and their corresponding components to the given world.
+    ///
+    /// This method will return an `SceneSpawnError` if either the type of a
+    /// component or the component itself hasn't been registered.
```

Comment:
To be more precise, the two errors cases are when a type is not registered and when a type doesn't reflect the Component trait.